test/common_framework.py

```diff
@@ -183,18 +183,18 @@ def setUpClassPost(cls):
                     major, minor = release().split('.')[0:2]
                     if not (major == '6' and minor in ['11', '12', '13']):
                         if environ.get("NO_KVM", '0') == '1':
-                            print("\nUsable KVM found but NO_KVM=1 - Disabling\n")
+                            print(" \nUsable KVM found but NO_KVM=1 - Disabling", file=stderr, flush=True)
```

Comment:
I don't understand, do
you really need trailing
space here?

Reply:
Its all confusing, I was
telling about the trailing
spaces that are actually
not there. Silly github
viewer just wraps the
lines that way.